Extend Third-party licenses topics to contain info about all packages (#4087)
magento/devdocs: _includes/release-notes/packages.md

> ## Magento packages

Reviewer: We need some introductory text before this heading, otherwise, we end up with the H1 topic title immediately followed by an H2 heading. It's bad practice.

Author: agree

> PHP packages used within Magento application are managed by Composer.

Reviewer: ```suggestion
The Magento application uses Composer to manage PHP packages.
```

> `composer.json` file declares the list of packages, whereas `composer.lock` stores complete list of the packages (a full version of each package and its dependencies) used to build a release version of the Magento application. This list of packages from `composer.lock` for the latest {{ edition }} {{page.guide_version}} release is presented in the tables below.

Reviewer: ```suggestion
The `composer.json` file declares the list of packages, whereas the `composer.lock` file stores a complete list of the packages (a full version of each package and its dependencies) used to build a release version of the Magento application. The following tables list packages from the `composer.lock` file for {{ edition }} {{page.guide_version}}.
```
